src/CalcViewModel/Common/NavCategory.cpp

```diff
@@ -276,6 +278,13 @@ static const list<NavCategoryInitializer> s_categoryManifest = [] {
     return res;
 }();
 
+void NavCategory::InitializeCategoryManifest(User ^ user)
+{
+    auto navCatInit = s_categoryManifest.begin();
+    std::advance(navCatInit, 2);
```

Comment:
Is it possible to not hardcode the index and detect which entry is for Graphing Mode? I can see this easily breaking in the future if someone updates the s_categoryManifest but doesn't update this code.

Reply:
Resolved